drivers/hmis_csv_twenty_twenty_four/config/routes.rb

```diff
@@ -0,0 +1,12 @@
+###
+# Copyright 2016 - 2023 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+BostonHmis::Application.routes.draw do
+  scope :hmis_csv_twenty_twenty_four do
+    # TODO
+    # get '/my_path', to: 'hmis_csv_twenty_twenty_four/my_controller'
```

Comment:
Is this really a TODO, or just left over from the driver generation?

Reply:
Just the default from the driver generator.